packages/cli/src/lib/snapshot/reportViewer.ts

```diff
@@ -6,8 +6,22 @@ import {
 import {cli} from 'cli-ux';
 import {bgHex, green, yellow, red, bold, italic} from 'chalk';
 
+export type ReportViewerOperationsToDisplay = {
+  [operation in keyof ResourceSnapshotsReportOperationModel]: boolean;
+};
+
 export class ReportViewer {
+  public static defaultOperationsToDisplay: ReportViewerOperationsToDisplay = {
+    resourcesCreated: true,
+    resourcesDeleted: true,
+    resourcesInError: true,
+    resourcesRecreated: true,
+    resourcesUnchanged: true,
+    resourcesUpdated: true,
+  };
+
```

Comment:
Here, I do not understand really the true/false 🤔 

I think the presence or lack thereof of the key (e.g. `resourcesCreated`) would work.

With this, you could do something like:
```suggestion
export type ReportViewerOperationsToDisplay = keyof ResourcesSnapshotsReportOperationModel //maybe not exactly that, but something similar.

export class ReportViewer {
  public static defaultOperationsToDisplay: ReportViewerOperationsToDisplay[] = [
    'resourcesCreated',
    'resourcesDeleted',
    'resourcesInError',
    'resourcesRecreated',
    'resourcesUnchanged',
    'resourcesUpdated',
  };
```

Reply:
This would of require to change your things to an array overall.

Reply:
I'm fine with that